portal/src/main/webapp/js/src/load-frontend.js

```diff
@@ -18,6 +25,7 @@ window.loadReactApp = function(config) {
                 '<div class="alert alert-warning">' +
                 '<button type="button" class="close" data-dismiss="alert">&times;</button>' +
                 'cbioportal-frontend dev mode, using localhost:3000' +
+                '&nbsp;<a href="#" onclick="clearDevState()">clear</a>' +
```

Comment:
this is a good idea. while you're at it, do you want to add something in here that looks for a URL parameter "localdev" or "heroku" and sets it automatically?